lib/facia/src/lib/facia-models.ts

```diff
@@ -0,0 +1,77 @@
+import { z } from "zod";
+
+//This should mirror FeastAppModel in Facia backend
+export const RecipeIdentifier = z.object({
+  id: z.string()
+})
+
+export type RecipeIdentifier = z.infer<typeof RecipeIdentifier>;
```

Comment:
I'm not familiar with what `zod` does or how, but will it be confusing to others in future if there are multiple exports with the same name?

Reply:
I guess what I _meant_ to ask was is it necessary to export both the `const` and the `type` definitions? Would we be importing `type`s primarily? Will read up in case I can answer this for myself :)

Reply:
`zod` is a library that does runtime JSON validation for Typescript - it's been adopted in CoPro to ensure that the json objects really do match the type signatures so we are using it here as well!

This is actually the recommended way of using it, despite it looking a bit weird.  The reason you have both a `const` and a `type` with the same name is so that you can do e.g. `RecipeIdentifier.parse(untypedJsonObject)` and get a `RecipeIdentifier` instance from it. So it kinda looks like the Scala companion-object pattern from the POV of the caller